sources/packages/backend/apps/db-migrations/src/sql/Types/Create-disbursement-overaward-origin-types.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TYPE sims.disbursement_overaward_origin_types AS ENUM (
+    'Reassessment overaward',
+    'Award value adjusted',
+    'Pending award cancelled',
+    'Manually entered',
+    'Imported'
```

Comment:
Is imported means coming from legacy?

Reply:
Imported means it was imported 😄 
Yes, the idea is to have the SFAS values imported into the table upon student creation or during the SFAS integration.
The type is there but it is not part of this PR.